server/sonar-server/src/main/java/org/sonar/server/batch/IssuesAction.java

```diff
@@ -40,6 +35,12 @@
 import org.sonar.server.plugins.MimeTypes;
 import org.sonar.server.user.UserSession;
 
+import java.io.IOException;
```

Comment:
I mean that you have diff with imports, which shouldn't be the case. They're grouped differently than the last commiter